includes/collection/class-interactions.php

```diff
@@ -248,10 +248,6 @@ public static function activity_to_comment( $activity ) {
 
 		$url = object_to_uri( $actor['url'] ?? $actor['id'] );
 
-		if ( ! $url ) {
-			$url = object_to_uri( $actor['id'] );
-		}
```

Comment:
I recon it is probably fine to keep?

The fallback in line 249 is for the case where `url` is null or undefined, the fallback here for the case when `$actor['url']` doesn't resolve to a url, is how I'm reading it.